docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
-sphinx==4.0.1
+sphinx==4.2.0
```

Comment:
sphinx 4.2.0 is the first version that seems to work with Python 3.10, see discussion [here](https://github.com/sphinx-doc/sphinx/issues/9562)